packages/edit-site/src/components/page-pages/index.js

```diff
@@ -200,6 +197,10 @@ export default function PagePages() {
 		[ postStatuses, authors ]
 	);
 
+	const filters = useMemo( () => [
```

Comment:
To be honest, I'm not sure I'd consider "search" a filter at all. It may be its own thing.

Reply:
https://github.com/WordPress/gutenberg/pull/55722